docs/useListController.md

```diff
@@ -43,11 +43,12 @@ const MyList = () => {
 * [`queryOptions`](./List.md#queryoptions): react-query options for the useQuery call
 * [`resource`](./List.md#resource): resource name, e.g. 'posts' ; defaults to the current resource context
 * [`sort`](./List.md#sort-default-sort-field--order), current sort value, e.g. { field: 'published_at', order: 'DESC' }
+* [`storeKey`](#storeKey): key used to differenciate the list from another sharing the same resource, in store managed states
```

Comment:
```suggestion
* [`storeKey`](./List.md#storeKey): key used to differenciate the list from another sharing the same resource, in store managed states
```

Not 100% sure this is required, but other list items do it like this so you should probably do it the same way